www/deprecation_policy.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: contribute
+title: Apache Cordova Deprecation Policy
+---
+
+# Deprecation and Archiving Policy
+
+Apache Cordova has been around quite some time, having been started back in August 2008. As it consists of many individual "parts", some of those stopped being useful for any number of reasons. This "Deprecation and Archiving Policy" describes why, when and how projects can be deprecated and archived - and what that actually means.
+
+## Deprecation Policy
+
+When a piece of Apache Cordova stops being useful or necessary we vote on deprecating it on our [dev mailing list](/contact/).
+
+### Reasons for Deprecation
+
+Common reasons for deprecation of a piece of Apache Cordova are:
+
+- General availability of a web standard that achieves the same thing
+- A better replacement is available somewhere else
+- Usage of a platform declined to a non relevant level
+- A platform was deprecated itself by its producer
+- ...
+
+### Meaning of Deprecation
+
+When we deprecate something it means no more work will be done on it. We make the deprecation clearly visible in the project's `README.md` (via a "Deprecation Notice" which might also include a reason and links to alternatives if available) and repository description.
```

Comment:
- I would add an explicit note to the end of the first sentence that we would no longer support the project and there is a risk of security issues that will never be fixed.
- I would turn part of the second sentence into a bullet list and include an item in the middle that the project would be set to "archived" state on GitHub.

Reply:
I don't fully follow. Can you just comment some text as a suggestion please?

Reply:
I guess you mean that you do not fully follow my second point. For clarity I will give an idea how I would rework the first paragraph, in a rough form:

> When we deprecate a component it means no more work will be done on it and it will no longer be supported. We make the deprecation clearly visible as follows:
> - mark the component as deprecated in the project's `README.md` (via a "Deprecation Notice")
> - mark the component as deprecated in the repository description.
> - add links to alternatives if available to the "deprecation notice" in `README.md`
> - set the repository to archived state as described in [link to archived section below]